bundles/org.openhab.binding.elroconnects/README.md

```diff
@@ -0,0 +1,190 @@
+# ElroConnects Binding
+
+The ElroConnects binding provides integration with the [Elro Connects](https://www.elro.eu/en/smart-home) smart home system.
+
+The system uses a Wifi Hub (K1 Connector) to enable communication with various smart home devices.
+The devices communicate with the hub using 868MHz RF.
+The binding only communicates with the Elro Connects system and K1 Connector using UDP in the local network.
+
+The binding exposes the devices' status and controls to openHAB.
+The K1 connector itself allows setting up scenes through a mobile application.
+The binding supports selecting a specific scene.
+
+Many of the sensor devices are battery powered.
+
+## Supported Things
+
+The Elro Connects supported device types are:
+
+* K1 connector hub: `connector`
+* Smoke detector: `smokealarm`
+* Carbon monoxide detector: `coalarm`
+* Heat detector: `heatalarm`
+* Water detector: `wateralarm`
+* Windows/door contact: `entrysensor`
+* Motion detector: `motionsensor`
+* Temperature and humidity monitor: `temperaturesensor`
+* Plug-in switch: `powersocket`
+
+The `connector` is the bridge thing.
+All other things are connected to the bridge.
+
+Testing was only done with smoke and water detectors connected to a K1 connector.
+The firmware version of the K1 connector was 2.0.3.30 at time of test.
+Older versions of the firmware are known to have differences in the communication protocol.
+
+## Discovery
+
+The K1 connector `connector` cannot be auto-discovered.
+Once the bridge thing representing the K1 connector is correctly set up and online, discovery will allow discovering all devices connected to the K1 connector.
+
+The communication between the hub and devices may be disturbed.
```

Comment:
What can disturb them? perhaps re-word this if the intension is to point out that discovery may not work if the device is out of reliable RF range.